app/src/main/java/com/hover/stax/utils/Constants.kt

```diff
@@ -28,6 +28,7 @@ object Constants {
     const val LANGUAGE_CHECK = "Language"
     const val PERM_ACTIVITY = "com.hover.stax.permissions.PermissionsActivity"
     const val TRANSFERRED = "TRANSFERRED"
```

Comment:
Constant name and value should be "TRANSFER" so as not to imply that the transfer definitely happened

Reply:
Actually, the instances where this constant is being used implies that the transfer definitely happened